<launch>
    <node pkg="robot_localization" type="ekf_localization_node" name="ekf_se_odom">
      <rosparam command="load" file="$(find husky_manager)/config/localization1.yaml" />
      <remap from="odometry/filtered" to="odometry/dead_reckoning"/>
    </node>

    <node pkg="robot_localization" type="ekf_localization_node" name="ekf_se_map">
      <rosparam command="load" file="$(find husky_manager)/config/localization2.yaml" />
      <remap from="tf" to="tf_redundant"/>
      <remap from="odometry/filtered" to="odometry/combined"/>
    </node>

    <node pkg="robot_localization" type="navsat_transform_node" name="navsat_transform_node" output="screen">
      <rosparam command="load" file="$(find husky_manager)/config/navsat_transform.yaml" /> 
      <!--Placeholders for input remapping. Set your topic names as the "to" values.  -->
      <!-- <remap from="imu/data" to=""/> -->
      <remap from="odometry/filtered" to="ekf/odom_filtered"/>
      <remap from="gps/fix" to="gnss/fix"/>
      <remap from="gps/filtered" to="navsat/gps_filtered"/>
      <remap from="odometry/gps" to="odometry/gps_aided"/>
    </node>
</launch>
